src/api/controller/api/job.js

```diff
@@ -1,7 +1,10 @@
 import Koa from 'koa';
 import route from 'koa-route';
 import koaBodyParser from 'koa-bodyparser';
-import { enrichmentQueue } from '../../workers/enrichment';
+import { enrichmentQueue, ENRICHMENT_QUEUE } from '../../workers/enrichment';
```

Comment:
Maybe not related to this PR but there's a naming issue. It's either publisher/enricher or publishment/enrichment